features-json/shadowdom.json

```diff
@@ -1,5 +1,5 @@
 {
-  "title":"Shadow DOM",
+  "title":"Shadow DOM v0",
   "description":"Method of establishing and maintaining functional boundaries between DOM trees and how these trees interact with each other within a document, thus enabling better functional encapsulation within the DOM.",
   "spec":"http://www.w3.org/TR/shadow-dom/",
```

Comment:
v1 is being merged into the HTML living standard, but that's a bit of a mess right now. I'd rather link here (which has all the updates)